src/components/dataset-api-doc/api-example.js

```diff
@@ -1,14 +1,14 @@
 import './api-example.scss'
 import FilterNoneIcon from '@material-ui/icons/FilterNone'
 
-export default ({ title, descriptionHtml, action, url, params, examples }) => {
+export default ({ title, descriptionHtml, action, url, params, examples, testId='' }) => {
```

Comment:
I am strongly against changing the function signature of production code to simply support testing purposes.

Reply:
done